docs/source/changelog.rst

```diff
@@ -1,6 +1,20 @@
 Changelog
 =========
 
+0.15.3 / 2017-09-04
+-------------------
+
+Array
++++++
+
+-  Added `*_like` array creation functions (:pr:`2640`)
+-  Added support for masked arrays (:pr:`2301`)
+
+DataFrame
++++++++++
+
+-  Added user defined aggregations (:pr:`2344`)
+
```

Comment:
The changelog makes a guess at which are "significant" changes since 0.15.2 (which is quite recent).
The full list of relevant merge commits is:

$ git log --oneline 94b0c251~..master | grep  "#[0-9]*"
 *   b1ff8260 WIP: Add user defined aggregations (#2344)
 *   b1c5f1b2 [docs] new cheatsheet (#2649)
 *   dd0802fb Masked arrays (#2301)
 *   db7bee4c Indexing with an unsigned integer array (#2647)
 *   d32e8b7b ENH: Allow the groupby by param to handle columns and index levels (#2636)
 *   358c6098 update copyright date (#2642)
 *   4cf62a34 python setup.py test runs py.test (#2641)
 *   da24ff52 Avoid using operator.itemgetter in dask.dataframe (#2638)
 *   bed713f3 Add `*_like` array creation functions (#2640)
 *   55067b20 Consistent slicing names (#2601)
 *   e1e3d157 Replace Continuum Analytics with Anaconda Inc. (#2631)
 *   3cd0dc55 Implement Series.str[index] (#2634)
 *   1a24af47 Support complex data with vnorm (#2621)
 *   94b0c251 bump version to 0.15.2 (#2627)

Please advise if I skimmed over something important !